client/app/startup/registration.js

```diff
@@ -13,6 +13,7 @@ import { Header } from '../components/Header';
 import { HeaderProfile } from '../components/Header/HeaderProfile';
 import { Resource } from '../components/Resource';
 import { Tag } from '../components/Tag';
+import { Bullet } from '../components/Bullet';
```

Comment:
You can remove this since the component isn't being used in any Rails views yet.